hls4ml/.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__/
```

Comment:
maybe add this to a top-level `.gitignore`

Reply:
Done.